collector/mountstats_linux.go

```diff
@@ -266,6 +286,129 @@ func NewMountStatsCollector(logger log.Logger) (Collector, error) {
 			nil,
 		),
 
+		NFSTransportReadChunkTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_read_chunk_total"),
+			"Total number of read chunk accessed when sending a call.",
+			labels,
+			nil,
+		),
+
+		NFSTransportWriteChunkTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_write_chunk_total"),
+			"Total number of write chunk accessed when sending a call.",
+			labels,
+			nil,
+		),
+
+		NFSTransportReplyChunkTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_reply_chunk_total"),
+			"Total number of reply chunk accessed when sending a call.",
+			labels,
+			nil,
+		),
+
+		NFSTransportRdmaRequestTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_rdma_requests"),
+			"Total number of rdma request accessed when sending a call.",
+			labels,
+			nil,
+		),
+
+		NFSTransportPullupCopyTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_pullup_copy_total"),
+			"Total number of pullup copy when rarely accessed error counters.",
+			labels,
+			nil,
+		),
+
+		NFSTransportHardwayRegisterTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_hardway_register_total"),
+			"Total number of hardway register when rarely accessed error counters.",
+			labels,
+			nil,
+		),
+
+		NFSTransportFailedMarshalTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_failed_marshal_total"),
+			"Total number of failed marshal when rarely accessed error counters.",
+			labels,
+			nil,
+		),
+
+		NFSTransportBadReplyTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_bad_reply_total"),
+			"Total number of bad reply when rarely accessed error counters.",
```

Comment:
Same here but also not sure what this sentence exactly means.

Reply:
Thanks for your review.
Change to `transport_bad_reply_requests_total`.

docs: https://www.ietf.org/archive/id/draft-ietf-nfsv4-rpcrdma-03.txt